catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java

```diff
@@ -188,14 +209,24 @@ public void exitQuery(BqsParser.QueryContext ctx) {
         List<Filter> filters = filterBy.get(operHash);
 
         if (filters != null && !filters.isEmpty()) {
-          if (currFilter != null) {
+          if (isValidFilter(currFilter)) {
+            // if (currFilter != null) {
             filters.add(currFilter);
           }
           currFilter = filterBuilder.anyOf(filters);
         }
       }
     }
 
+    protected boolean isValidFilter(Filter filter) {
+      boolean ret = false;
```

Comment:
what does `ret` mean?

Reply:
Changed to `valid` for clarity

Reply:
return value - I was being lazy - valid is a much better name